### PR TITLE
LPD-40224

### PR DESCRIPTION
**Pull request by daiazius**

### CODE REVIEW

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
ci:test:sf

File: modules/test/playwright/tests/portal-security-audit-web/auditExport.spec.ts [hunk 1]
```diff
@@ -41,58 +45,96 @@ const fields = [
...
+
+		const download = await downloadPromise;
+
+		const filePath = getTempDir() + download.suggestedFilename();
+
+		await download.saveAs(filePath);
+
+		const content = readFileSync(filePath, 'utf8');
+
+		const regex = '/\b(ADD|ASSIGN|UPDATE)\b/g';
```
Comment (rafaprax) [inline modules/test/playwright/tests/portal-security-audit-web/auditExport.spec.ts:129]: Hey @daiazius, please inline the `regex` variable.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:security-playwright

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:security-playwright

**Comment by ChrisKian** [comment]
ci:test:sf

**Comment by ChrisKian** [comment]
ci:test:security-playwright

**Comment by rafaprax** [comment]
ci:test:security-playwright

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:security-playwright

**Pull request closed without merging**